### ISSUE TITLE
[BEGINNER QUESTION] How to do image swap on Collab

### ISSUE THREAD

**Issue by de-luxe110** [neuralchen/SimSwap#183]
I have been using Simswap for video swapping and it has been great. I am using the Collab since I don't have the necessary hardware to run the program on my PC. I am also just a hobbyist and doesn't understand most of the coding made on the Collab but I kinda understood what codes to change to make it work. So my question is how can I run an image swap on the Collab because I tried pasting in the code on the Inference page but it's just giving me errors. It would be better if you could explain it to my simply too in layman terms so that I could understand what I'm doing. 

Thank you and may Simswap continue to prosper!

**Comment by Gitgutx** [comment]
Look at my comment on issue #185 for how to use images on colab.

Look at issue #162 if you want to use the 512 version on colab (not recommended as of now, 224 usually look better, you may want to use methods like GFPGAN or GPEN to improve them).
